whisper/whisper/load_models.py

```diff
@@ -191,8 +194,51 @@ def convert_rblock(model, rules):
 
 
 def load_model(
-    name: str,
+    name_or_path: str,
     download_root: str = None,
     dtype: mx.Dtype = mx.float32,
 ) -> whisper.Whisper:
-    return torch_to_mlx(load_torch_model(name, download_root), dtype)
+    if name_or_path in _MODELS:
+        print(f"[INFO] Loading and converting {name_or_path} model")
+        return torch_to_mlx(load_torch_model(name_or_path, download_root), dtype)
```

Comment:
Do you think it would be better to move this and all conversion utilities (`_download`, `load_torch_model`, `convert`, `torch_to_mlx`) into convert.py, to separate conversion and loading, as has been done for other models?

Reply:
Yea, I think that would probably be cleaner. (But I'm ok merging this PR as is as well). I'll leave it to you to decide on that.

Reply:
let me make it cleaner, can also encourage users to convert the model once and for all, and upload more models to Hugging Face :)

Reply:
Awesome!! Would love to see some models on Hugging Face. The MLX community would be a great place to put them: https://huggingface.co/mlx-community